### ISSUE TITLE
🐛 Bug Report: task viewer logs not resizable

### ISSUE THREAD

**Issue by tstollin** [backstage/backstage#25875]
### 📜 Description

The resizable box containing the task logs in the task viewer, which was introduced with https://github.com/backstage/backstage/pull/25495, is not resizable. 
Because of this, the box height is  fixed to 240p and only shows 7 lines, which highly limits readability.

### 👍 Expected behavior

The box containing the task logs should be resizable as shown in https://github.com/backstage/backstage/pull/25495. Alternatively, an option to configure and fix the box height would work for us.

### 👎 Actual Behavior with Screenshots

The draggable resize icon (corner icon in the lower right corner of the box) is missing:
<img width="840" alt="image" src="https://github.com/user-attachments/assets/2af1c6d4-0df2-4ace-bbd1-8e876367e9d1">

The span, which is used as the draggable element, is part of the html of the page, but since it has no content, it is invisible:
![image](https://github.com/user-attachments/assets/577f16ee-1c51-4ad8-b531-44b325d89f82)

After manually editing the HTML and inserting some content, the element becomes visible and is draggable. WIth dragging the element, the box can be resized:
<img width="1663" alt="image" src="https://github.com/user-attachments/assets/f2000d53-7c4f-400c-9076-2893cd188dc4">

### 👟 Reproduction steps

If at least one task has been run before:

1. Open the task list /scaffolder/tasks (not sure about the default route, our routes differ)
2. Select any task 
3. Select the three dots on the upper right hand side
4. Select "Show Logs"
5. See the log output

If not:

1. Run a task 
2. See the log output

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

No, I don't have time to work on this right now

**Comment by benjdlambert** [comment]
Woops! @Zaperex do you have time to take a look at this? Otherwise open to any contributions here :pray:

**Comment by fedy97** [comment]
I am not reproducing this, I am aligned with master

<img width="1187" alt="image" src="https://github.com/user-attachments/assets/a54535c1-ecce-4373-89f3-7d50a2a5fdef">

**Comment by drodil** [comment]
I have also 1.29.1 and the resizing is working just fine. What I would like to see is maybe to make the default size a bit larger as those couple of lines are not usually enough to get even a small idea about what has happened and requires user interaction always.

**Comment by benjdlambert** [comment]
Thinking that we should probably create another issue and followup PR for the default sizes, and close this ticket if it's actually fine in the latest release. @tstollin can you check if it's working on the latest release? `yarn backstage-cli versions:bump`

**Comment by tstollin** [comment]
We were already on 1.29.2 when the problem occured.
After the comment from @fedy97 I did some further investigations: a missing `react-resizable/css/styles.css` was causing the issue. Unfortunately, I did not understand why it was missing as we are using the scaffolder plugin as is. 

As a workaround, I fixed it by adding react-resizable as a dependency to our app and importing the the css in our App.tsx manually.

Currently, I'm fine with your suggestion @benjdlambert about closing this issue, but I would be very grateful if someone could give me a hint why the css could be missing.
